client/src/pages/AddFood.js

```diff
@@ -77,16 +77,22 @@ function AddFood() {
 }
 
 const Form = styled.form`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-49%, -49%);
```

Comment:
How do you feel about using flexbox to center this form instead of absolute positioning?